analyzers/src/SonarAnalyzer.VisualBasic/Rules/WeakSslTlsProtocols.cs

```diff
@@ -34,11 +34,34 @@ public sealed class WeakSslTlsProtocols : WeakSslTlsProtocolsBase<SyntaxKind, Id
     {
         protected override GeneratedCodeRecognizer GeneratedCodeRecognizer { get; } = VisualBasicGeneratedCodeRecognizer.Instance;
 
-        protected override SyntaxKind SyntaxKind { get; } = SyntaxKind.IdentifierName;
+        protected override SyntaxKind SyntaxKindOfInterest { get; } = SyntaxKind.IdentifierName;
 
         protected override DiagnosticDescriptor Rule { get; } = DiagnosticDescriptorBuilder.GetDescriptor(DiagnosticId, MessageFormat, RspecStrings.ResourceManager);
 
         protected override string GetIdentifierText(IdentifierNameSyntax identifierNameSyntax) =>
             identifierNameSyntax.Identifier.Text;
+
+        protected override bool IsNodeOfInterest(SyntaxNode node)
+        {
+            if (!(node.Parent is MemberAccessExpressionSyntax))
+            {
+                return false;
+            }
+
+            var current = node;
```

Comment:
I think we can replace `parent` with `current.Parent` in order to have just a local variable and skip some assignments.